### PR TITLE
#162751765 Create endpoints for creating meetup

### PR DESCRIPTION
**Pull request by innocentEdosa**
#### What does this PR do?
Setup POST/api/v1/meetups endpoint

#### Description of Task to be completed?
create an express server
create meetup model
create appropriate routes
create needed controlleer

#### How should this be manually tested?
clone the repo //#endregion
checkout to ft-createMeetup-endpoint-163047983
open terminal
npm run serve
test with postman

#### Any background context you want to provide?
This endpoints enables the creation of new meetups

#### What are the relevant pivotal tracker stories?
163047983

#### Screenshots
no screenshots

#### Questions
can this be merged?

### CODE REVIEW

File: dist/models/meetup.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
+'use strict';
+
+Object.defineProperty(exports, "__esModule", {
+  value: true
+});
+
+var _createClass = function () { function defineProperties(target, props) { for (var i = 0; i < props.length; i++) { var descriptor = props[i]; descriptor.enumerable = descriptor.enumerable || false; descriptor.configurable = true; if ("value" in descriptor) descriptor.writable = true; Object.defineProperty(target, descriptor.key, descriptor); } } return function (Constructor, protoProps, staticProps) { if (protoProps) defineProperties(Constructor.prototype, protoProps); if (staticProps) defineProperties(Constructor, staticProps); return Constructor; }; }();
```
Comment (chukwuemekachm) [inline dist/models/meetup.js]: Why are you committing your transpiled javascript files?

**Review by chukwuemekachm — CHANGES_REQUESTED** [review]
Nice. Please review your comments.

File: dist/models/meetup.js (hunk 1 above)
Comment (innocentEdosa) [reply dist/models/meetup.js]: My mistake
sorry about that 
would fix now
thanks

File: server/app.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import express from 'express';
+
+// import json body parser for express
```
Comment (chukwuemekachm) [inline server/app.js]: These comments are not required. They are redundant.
Please remove them.

File: server/controller/meetup.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import Meetup from '../models/meetup';
+
+
+// this controller handles the creation of new meetups
+export default class meetupController {
```
Comment (chukwuemekachm) [inline server/controller/meetup.js:6]: Change this to a [JSDOCS](http://usejsdoc.org/) comment.

File: server/models/meetup.js [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+  location: 'aduwawa',
+  images: 'this is the imgae url',
+  topic: 'This is the best meetup ever ',
+  happeningOn: new Date(2019, 2, 1),
+  tags: ['business', 'entertianment'],
+  description: 'this is a very short description of this very second meetup',
+  noOfQuestions: 2,
+}];
+
+// create a class to handle the meetup array
```
Comment (chukwuemekachm) [inline server/models/meetup.js:3]: Change this to a [JSDOCS](http://usejsdoc.org/) comment.

File: server/models/meetup.js [hunk 5]
```diff
@@ -0,0 +1,72 @@
+const meetup = [{
```
Comment (chukwuemekachm) [inline server/models/meetup.js]: Abstract your dataset into a different file.

File: server/route/index.js [hunk 6]
```diff
@@ -0,0 +1,13 @@
+import { Router } from 'express';
+
+import meetupRouter from './meetup';
+
+
+const router = Router();
+
+
+//  import the meetup controller
```
Comment (chukwuemekachm) [inline server/route/index.js]: This comment is redundant.

**Review by chukwuemekachm — CHANGES_REQUESTED** [review]
Good job @innocentEdosa.
Please attend to these comments.

File: server/models/meetup.js (hunk 4 above)
Comment (innocentEdosa) [reply server/models/meetup.js:3]: Hello chima
I have checked comment to jsdocs now
Thanks

**Review by chukwuemekachm — APPROVED** [review]
Nice @innocentEdosa.

**Pull request merged**